site/src/api/queries/workspaceBuilds.ts

```diff
@@ -58,17 +58,9 @@ export const infiniteWorkspaceBuilds = (
 };
 
 // We use readyAgentsCount to invalidate the query when an agent connects
-export const workspaceBuildTimings = (
-	workspaceBuildId: string,
-	readyAgentsCount: number,
-) => {
+export const workspaceBuildTimings = (workspaceBuildId: string) => {
 	return {
-		queryKey: [
-			"workspaceBuilds",
-			workspaceBuildId,
-			"timings",
-			{ readyAgentsCount },
```

Comment:
What was the `readyAgentCount` value doing before? I see that we were only using it for the query key, and now it's gone

Reply:
It was to refetch the query when a new agent was marked as ready but it was very buggy and a bad decision on my end. The current refetch logic makes more sense and it is easier to follow.